packages/released/coq-menhirlib/coq-menhirlib.20190613/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "A support library for verified Coq parsers produced by Menhir"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "Jacques-Henri Jourdan <jacques-henri.jourdan@lri.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/coq-menhirlib"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/menhir.git"
+bug-reports: "jacques-henri.jourdan@lri.fr"
+build: [
+  [make "-C" "coq-menhirlib" "-j%{jobs}%"]
+]
+install: [
+  [make "-C" "coq-menhirlib" "install"]
+]
+remove: [
+  [make "-C" "coq-menhirlib" "uninstall"]
+]
+depends: [
+  "coq" { >= "8.6" }
+  "menhir" { = "20190613" }
```

Comment:
According to my local tests, this package does in fact not depend on `menhir`. The Coq files can be compiled (and used) standalone. So please omit this dependency.

Reply:
Indeed, the package does not depend on `menhir`. However, most people who use `coq-menhirlib` are likely to use `menhir` as well. Furthermore, we would like to express the constraint that `menhir`, if present, must be at version `20190613`, so as to prevent people from installing inconsistent versions of the two packages.

Reply:
Perhaps this can be expressed as
```
depopts: "menhir"
conflicts: [
  "menhir" { != "20190613"}
]
```

Reply:
Thanks, I have added a `conflicts:` field. I don't think the `depopts` field is needed, as `coq-menhirlib` itself is insensitive to the presence or absence of `menhir`.